packages/graphback-codegen-resolvers/src/templates/LayeredResolverTemplates.ts

```diff
@@ -0,0 +1,111 @@
+// TODO subscription support
+// TODO support for the database type instead of hardcoded lowercase
+export const createTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
+  return `${fieldName}: (_, args, context) => {
+      validateContext(context)
+      return context.crudService.create("${typeName.toLowerCase()}", args.input, context);
+    }`
+}
+
+export const updateTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
+  return `${fieldName}: (_, args, context) => {
+      validateContext(context)
+      return context.crudService.update("${typeName.toLowerCase()}", args.id, args.input, context);
+    }`
+}
+
+export const deleteTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
+  return `${fieldName}: (_, args, context) => {
+      validateContext(context)
+      return context.crudService.delete("${typeName.toLowerCase()}", args.id, args.input, context);
+    }`
+}
+
+export const findAllTemplate = (fieldName: string, typeName: string): string => {
+  return `${fieldName}: (_, args, context) => {
+      validateContext(context)
+      return context.crudService.findAll("${typeName.toLowerCase()}", context);
+    }`
+}
+
+export const findTemplate = (fieldName: string, typeName: string, ): string => {
+  return `${fieldName}: (_, args, context) => {
+      validateContext(context)
+      return context.crudService.findBy("${typeName.toLowerCase()}", args.fields, context);
+    }`
+}
+
+export const newSub = (typeName: string): string => {
+  return `new${typeName}: {
+      subscribe: (_: any, __: any, context: GraphQLContext) => {
+        validateContext(context)
+        return context.crudService.subscribeToCreate("${typeName.toLowerCase()}", context);
+      }
+    }`
+}
+
+export const updatedSub = (typeName: string): string => {
+  return `updated${typeName}: {
+      subscribe  : (_: any, __: any, context: GraphQLContext) => {
```

Comment:
I know subscriptions are not working yet, but will they be fixed in separate PR?

Reply:
Yes. Subscriptions bring a lots of changes. I prefer to have small dedicated PR to fix them rather than overloading this PR - same with pagination and batching

Reply:
I wanted to make verification of PR possible, but this meant changing more than usual.